lib/rnaget/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ghcr.io/bento-platform/bento_base_image:python-debian-2024.07.09
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+RUN pip install --no-cache-dir "uvicorn[standard]==0.30.1"
+
+
+WORKDIR /tds
+
+COPY pyproject.toml .
+COPY poetry.lock .
+
+RUN poetry config virtualenvs.create false && \
+    poetry --no-cache install --without dev --no-root
+
+COPY transcriptomics_data_service transcriptomics_data_service
+COPY lib lib
+COPY entrypoint.bash .
+COPY run.bash .
+COPY LICENSE .
+COPY README.md .
+COPY create_db.sh .
```

Comment:
Can this be simplified? `COPY . /tds`?

Reply:
There are extra files like .git, tests, and other Docker files that shouldn't be included. I copied this from their Docker setup, so I'm not sure what their original purpose not including them.